core/node/state_keeper/src/keeper.rs

```diff
@@ -408,6 +405,18 @@ impl ZkSyncStateKeeper {
         Err(Error::Canceled)
     }
 
+    #[tracing::instrument(skip(updates_manager))]
```

Comment:
Nit: Please use `level = "debug"` here. Or you can remove the instrumentation altogether given that info about L2 block params is logged below anyway.

Reply:
Thanks, I'll just remove it